packages/nextjs/hooks/scaffold-eth/useScaffoldEventHistory.ts

```diff
@@ -45,7 +45,7 @@ export const useScaffoldEventHistory = <
   watch,
   enabled = true,
 }: UseScaffoldEventHistoryConfig<TContractName, TEventName, TBlockData, TTransactionData, TReceiptData>) => {
-  const [events, setEvents] = useState<any[]>();
+  const [events, setEvents] = useState<any[]>([]);
   const [isLoading, setIsLoading] = useState(false);
```

Comment:
should we change the default value to true?

Reply:
I think no, so we can distinguish when there is contract loading `deployedContractLoading`, and when we're reading events `isLoading`